tests/app/shared/issue/assignee/assignee.component.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { of } from 'rxjs';
+
+import { DebugElement } from '@angular/core';
+import { ComponentFixture, TestBed, async } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { By } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
+import { PhaseService } from '../../../../../src/app/core/services/phase.service';
+import { User, UserRole } from '../../../../../src/app/core/models/user.model';
+import { AssigneeComponent } from '../../../../../src/app/shared/issue/assignee/assignee.component';
+import { ErrorHandlingService } from '../../../../../src/app/core/services/error-handling.service';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../../constants/githubissue.constants';
+import { Issue } from '../../../../../src/app/core/models/issue.model';
+import { IssueService } from '../../../../../src/app/core/services/issue.service';
+import { MaterialModule } from '../../../../../src/app/shared/material.module';
+import { PermissionService } from '../../../../../src/app/core/services/permission.service';
+import { Team } from '../../../../../src/app/core/models/team.model';
+import { UserService } from '../../../../../src/app/core/services/user.service';
+import { Phase } from '../../../../../src/app/core/models/phase.model';
+
+describe('AssigneeComponent', () => {
+  let component: AssigneeComponent;
+  let debugElement: DebugElement;
+  let nativeElement: HTMLElement;
+  let fixture: ComponentFixture<AssigneeComponent>;
+  let dummyIssue: Issue;
+
+  const testStudent: User = {
+      loginId: 'testStudent',
+      role: UserRole.Student
+  };
+
+  const dummyTeam = new Team({
+      id: 'F09-2',
+      teamMembers: [testStudent]
```

Comment:
Can we have a few more team members? Will make the tests more realistic

Reply:
Sure! Then from which, will the current tests suffice? Or should we test for cases when we have > 1 assignee?

Reply:
The current tests will suffice

Reply:
Alright!